src/inject.js

```diff
@@ -136,6 +136,9 @@ const stringHandlers = {
 // set of class names)
 let alreadyInjected = {};
 
+// The number of classes that have been injected
+let classUIDCount = 0;
```

Comment:
The reason we use hashing in the first place is to ensure that rehydration will work: If we take an approach like this one (which we were using in the original implementation, see https://github.com/Khan/aphrodite/commit/f45f87d25f53644a360979479bf0c0b8a8ba1ff8), then the server and client might disagree on what the number should be for a given style (either because the code ran in a different order on the client or server, or because this is a file-local variable and the server keeps it around in between runs).

So, we definitely need to do hashing in here, an incrementing value isn't going to work. :(